docs/new-architecture-appendix.md

```diff
@@ -27,11 +27,31 @@ You may use the following table as a reference for which types are supported and
 
 You may also find it useful to refer to the JavaScript specifications for the core modules in React Native. These are located inside the `Libraries/` directory in the React Native repository.
 
-## II. Invoking the code-gen during development
+## II. TypeScript to Native Type Mapping
+
+You may use the following table as a reference for which types are supported and what they map to in each platform:
+
+| TypeScript Type                                | Nullable Support?                                        | Android (Java)                       | iOS                                                            | Note                                                                           |
```

Comment:
I'm not sure whether this table is correct or not. Who has a good knowledge of TypeScript to check this?